helm-chart/renku/values.yaml

```diff
@@ -1498,6 +1498,33 @@ dataService:
   trustedProxies:
     proxiesCount: 1
     realIpHeader: x-real-ip
+  # Build session images from code repositories with Shipwright
+  imageBuilders:
+    # Experimental feature: enable at your own risk!
+    # The feature depends on Shipwright (version >= 0.15.x) being already installed independently from the Renku Helm chart.
+    enabled: false
+    ## The container image prefix for images built from code
+    outputImagePrefix: harbor.dev.renku.ch/renku-build/
+    ## The run image used for Python + VSCodium images
+    vscodiumPythonRunImage: renku/renkulab-vscodium-python-runimage:ubuntu-59fcea4
+    ## The name of the BuildStrategy to use for image builds.
+    strategyName: renku-buildpacks
+    ## The name of the secret used to push images built from code.
+    pushSecretName: renku-build-docker-secret
+    ## The TTL for BuildRuns
+    buildRunRetentionAfterFailedSeconds: 86400
+    buildRunRetentionAfterSucceededSeconds: 86400
+    ## The timeout duration for image builds.
+    buildRunTimeoutSeconds: 3600
+    ## The node selector for BuildRuns
+    nodeSelector:
+      renku.io/node-purpose: image-build
+    ## The tolerations for BuildRuns
+    tolerations:
+      - effect: NoSchedule
+        key: renku.io/dedicated
+        operator: Equal
+        value: image-build
```

Comment:
should this be commented out to keep the value functional in simple deployments?

Reply:
This follows the current efforts from @aledegano to reduce the amount of custom values for CI/CD deployments. If you want to switch `enabled: true`, then having to adjust the values here is expected for now.